include/CoolPropLib.h

```diff
@@ -368,6 +368,38 @@ EXPORT_CODE void CONVENTION AbstractState_get_mole_fractions(const long handle,
 EXPORT_CODE void CONVENTION AbstractState_get_mole_fractions_satState(const long handle, const char* saturated_state, double* fractions,
                                                                       const long maxN, long* N, long* errcode, char* message_buffer,
                                                                       const long buffer_length);
+/**
+     * @brief Return the fugacity of the i-th component of the mixture.
+     * @param handle The integer handle for the state class stored in memory
+     * @param i i-th component of the mixture
+     * @param errcode The errorcode that is returned (0 = no error, !0 = error)
+     * @param message_buffer A buffer for the error code
+     * @param buffer_length The length of the buffer for the error code
+     * @return 
+     */
+EXPORT_CODE double CONVENTION AbstractState_get_fugacity(const long handle, const std::size_t i, long* errcode, char* message_buffer,
```

Comment:
everywhere you want long instead of std::size_t. size_t is not a standard C type

Reply:
the functions `fugacity_coefficient` and `fugacity` are defined to expect `std::size_t i`. Does it not have to be the same? 

from `AbstractState.cpp` 
```
double AbstractState::fugacity(std::size_t i)
double AbstractState::fugacity_coefficient(std::size_t i)
```

Reply:
No, the external interface should only use C datatypes, and cast C++ ones to C ones as needed

Reply:
Ok thanks for clarifying. I have made that change as you requested, so I will resolve this thread.